README.md

```diff
@@ -17,3 +17,4 @@ unt 切韻擬音 J | unt's _Qieyun_ Reconstruction J | `unt_j.js`
 推導普通話 | Extrapolated Putonghua | `putonghua.js`
 推導廣州音 | Extrapolated Cantonese | `gwongzau.js`
 綾香思考音系 | | `ayaka_v8.js`
+推導日語漢音 | Extrapolated Japanese Kanon | `japanese-kanon.js`
```

Comment:
Just call it 日語漢音 (or perhaps 日譯漢音 is better?), and the English name is _Kan-on_ (I am not sure whether the italic style is necessary here).

For the file name we can just call it `kan-on.js` (I am not sure whether the hyphen is necessary here).

Reply:
I won't prefer 日譯漢音 because it's a little weird and we don't call those 漢字音 a translation (譯). Also the fact that 日語漢音 & 日譯漢音 are both attested in academic papers, while 日語漢音 is used significantly more frequently.

Although 漢音 itself is kind of artificial, it is based on ancient usage. This process would never be “real”, it is in fact just a extrapolated model that can predict well on Japanese Kan-on. So I would like to still use the term “extrapolated” / 推導.

```suggestion
推導日語漢音 | Extrapolated Japanese Kan-on | `kan-on.js`
```

Reply:
Okay, I see!